### ISSUE TITLE
How to create a commit with multiple files and actions with java-api

### ISSUE THREAD

**Issue by welliu** [gmessner/gitlab4j-api#122]
I know the gitlab has this api[create-a-commit-with-multiple-files-and-actions](https://docs.gitlab.com/ee/api/commits.html#create-a-commit-with-multiple-files-and-actions), but I didnt  find this fuction in gitlab4j.

**Comment by gmessner** [comment]
Support for batch commit creation has been added to the CommitsApi and pushed to the master branch.  I have not fully tested the new functionality yet, will create unit tests and officially release later this week. 

If you could manually test and let me know how that goes I would appreciate it.  The methods to look at can be found in the CommitApi and are as follows:
```
public Commit createCommit(int projectId, String branch, String commitMessage, 
    String startBranch, String authorEmail, String authorName, List<CommitAction> actions)
public Commit createCommit(String project, String branch, String commitMessage,
    String startBranch, String authorEmail, String authorName, List<CommitAction> actions)
```

**Comment by gmessner** [comment]
This functionality has been released in version 4.7.11.  

Please let me know as soon as possible if you run into any issues.
